### PR TITLE
install ins. in hindi, fixed others Fixes #702

### LINKED ISSUE #702: Wrong Install Instructions
### Expected results
- The install instructions should be correct 🔥 
### Actual results
- Windows install instructions contain the command `ls` which should be `dir`

### Steps to reproduce the issue
- Edit the documentation 

## Would you like to work on this
Working.

### PR DESCRIPTION
**Pull request by YashKumarVerma**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request
-Added install instructions hindi
- Windows installation had command `ls` which is linux specific. replaced it with `dir`.

Fixes #702

### CODE REVIEW

**Review by nzec — APPROVED** [review]
`स्थानीय विकास सेटअप` 🤣 That sounds hilarious. Well, it looks good.

File: installation/installation_windows(hi).md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+### स्थानीय रूप से सेट करना
+1. इस रंजिश को फोर्क करें
+2. ओपन जीआईटी टर्मिनल (विंडोज़ में जीआईटी बाश), और अपने फोर्केड रेपो को क्लोन करें i.e. `जीआईटी क्लोन https://github.com/ <your_username> / gci17.fossasia.org.git` आप जिस निर्देशिका में चाहते हैं
+3. डायरेक्टरी को स्थानीय रिपॉजिटरी में बदलें i.e. `cd gci17.fossasia.org` अब `ls` टाइप करें, यह देखने के लिए कि क्या 'जेमफाइल' को देखा जा सकता है।
+4. निर्भरता स्थापित करने के लिए इसे टाइप करें `bundle install`
+5.  `gem install jekyll-sitemap` फिर `gem install jekyll-paginate`.(आवश्यक नहीं है, यदि यह पिछले चरण में स्थापित है।)
+6. अब जेफफ़ाइल खोलें और फिर '#' में हटा दें `#gem 'github-pages', group: :jekyll_plugins` और '#' में जोड़ें  `gem 'jekyll-sitemap'` 
+`gem 'jekyll-paginate'`. फ़ाइल को डिस्क में सहेजें
+7.यह टर्मिनल में लिखें `bundle exec jekyll serve `. आपको एक सर्वर पता दिखाई देगा.
+8. ओउ आपका सर्वर ऊपर है और चल रहा है Gci17.fossasia.org पृष्ठ देखने के लिए, `स्थानीयहोस्ट: 4000` पर जाएं.
```
Comment (harshit98) [inline installation/installation_windows(hi).md]: Gci17.fossasia.org -> gci17.fossasia.org

Comment (YashKumarVerma) [reply installation/installation_windows(hi).md]: fixed. 👍

File: installation/installation_macos(hi).md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+### jekyll स्थापना
+1. अपने टर्मिनल में 'मणि इंस्टॉल जेकइल बंडलर' दर्ज करें
+
+### स्थानीय रूप से सेट करना
+1. इस भंडार को फोर्क करें
+2. अपने वांछित भंडार को क्लोन करने के लिए git का प्रयोग करें, यानी git क्लोन। Http: //github.com/ <your_username> / gci17.fossasia.org.git
+3. डायरेक्टरी को स्थानीय रिपॉजिटरी में बदलें, यानी `सीडी gci17.fossasia.org` अब `ls` टाइप करें, यह देखने के लिए कि क्या 'जेमफाइल' को देखा जा सकता है।
+4. निर्भरताएं स्थापित करने के लिए `बंडल इंस्टॉल` टाइप करें
+5. टाइप करें `बंडल एंट जेकाइल सेवा` आपको एक सर्वर पता दिखाई देगा।
+6. अब आपका सर्वर ऊपर और चल रहा है। Gci17.fossasia.org पृष्ठ देखने के लिए, `स्थानीयहोस्ट: 4000` पर जाएं
```
Comment (akshatnitd) [inline installation/installation_macos(hi).md]: Change `Gci17.fossasia.org` to `gci17.fossasia.org`

File: installation/installation_linux(hi).md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+1. अपने टर्मिनल में 'मणि इंस्टॉल जेकइल बंडलर' दर्ज करें
+
+### स्थानीय रूप से सेट करना
+1. इस भंडार को फोर्क करें
+2. अपने वांछित भंडार को क्लोन करने के लिए git का प्रयोग करें, अर्थात।
+`जीआईटी क्लोन https://github.com/ <your_username> / gci17.fossasia.org.git`
+3. डायरेक्टरी को स्थानीय रिपॉजिटरी में बदलें, यानी `सीडी gci17.fossasia.org` अब `ls` टाइप करें, यह देखने के लिए कि क्या 'जेमफाइल' को देखा जा सकता है।
+4. निर्भरताएं स्थापित करने के लिए `बंडल इंस्टॉल` टाइप करें
+5. टाइप करें `बंडल एंट जेकाइल सेवा` आपको एक सर्वर पता दिखाई देगा।
+6. अब आपका सर्वर ऊपर और चल रहा है। Gci17.fossasia.org पृष्ठ देखने के लिए, `स्थानीयहोस्ट: 4000` पर जाएं
```
Comment (akshatnitd) [inline installation/installation_linux(hi).md]: Change `Gci17.fossasia.org` to `gci17.fossasia.org`

**Review by akshatnitd — CHANGES_REQUESTED** [review]

**Review by ParthS007 — APPROVED** [review]

**Pull request merged**